### PR TITLE
feat: added favicon, long logo and normal logo

### LINKED ISSUE #59: [Enhancement]: Add logo to website
### My idea

1. Add our logo from #33 to `static/` (both long and niemal variants)
2. Create favicon and add it also to same dir
4. Configure everything (make sure favicoon worka, display long logo in main page etc.)

### PR DESCRIPTION
**Pull request by toriishia**

### CODE REVIEW

File: core/templates/core/index.html [hunk 1]
```diff
@@ -27,6 +27,9 @@
     <canvas id="starback-bg" class="fixed -z-10 bg-gray-950"></canvas>
 
     <div class="relative flex flex-col items-center justify-center h-full text-center">
+        {% load static %}
+
+        <img src="{% static 'images/long_logo.svg' %}" class="w-1/2">
```
Comment (Norbiros) [inline core/templates/core/index.html]: `w-1/2` is relative to screen size. Use `h-60`.

**Review by Norbiros — CHANGES_REQUESTED** [review]
Almost looks good

**Review by Norbiros — APPROVED** [review]
LGTM!

**Review by Sanielia — APPROVED** [review]
merge now

**Pull request merged**